test/natalie/array_test.rb

```diff
@@ -1180,6 +1180,17 @@ def x.to_a
     end
   end
 
+  describe '#pack' do
+    describe 'c' do
+      fit 'handle bignums' do
```

Comment:
Just noticed this is focused @richardboehme :grimacing:

Reply:
Fixed in #266 😅